plugins/statistics.koplugin/main.lua

```diff
@@ -1447,42 +1449,59 @@ function ReaderStatistics:getCurrentStat()
     self.data.pages = self.view.document:getPageCount()
     total_time_book = tonumber(total_time_book)
     total_read_pages = tonumber(total_read_pages)
-    local time_to_read = self.view.state.page and ((self.data.pages - self.view.state.page) * self.avg_time) or 0
+
+    local current_page
+    local total_pages
+    local page_progress
+    if (self.view.document:hasHiddenFlows()) then
+        local flow = self.view.document:getPageFlow(self.view.state.page)
+        current_page = self.view.document:getPageNumberInFlow(self.view.state.page)
+        total_pages = self.view.document:getTotalPagesInFlow(flow)
+        if flow == 0 then
+            page_progress = ("%d // %d (%d%%)"):format(current_page, total_pages, Math.round(100*current_page/total_pages))
+        else
+            page_progress = ("[%d / %d]%d (%d%%)"):format(current_page, total_pages, flow, Math.round(100*current_page/total_pages))
+        end
+    else
+        current_page = self.view.state.page
+        total_pages = self.data.pages
+        page_progress = ("%d / %d (%d%%)"):format(current_page, total_pages, Math.round(100*current_page/total_pages))
+    end
+    local time_to_read = current_page and ((total_pages - current_page) * self.avg_time) or 0
     local estimate_days_to_read = math.ceil(time_to_read/(book_read_time/tonumber(total_days)))
-    local estimate_end_of_read_date = os.date("%Y-%m-%d", tonumber(now_ts + estimate_days_to_read * 86400))
+    local estimate_end_of_read_date = datetime.secondsToDate(tonumber(now_ts + estimate_days_to_read * 86400), true)
     local estimates_valid = time_to_read > 0 -- above values could be 'nan' and 'nil'
+
     local user_duration_format = G_reader_settings:readSetting("duration_format", "classic")
     return {
         -- Global statistics (may consider other books than current book)
-        -- since last resume
-        { _("Time spent reading this session"), datetime.secondsToClockDuration(user_duration_format, current_duration, false) },
-        { _("Pages read this session"), tonumber(current_pages) },
-        -- today
-        { _("Time spent reading today"), datetime.secondsToClockDuration(user_duration_format, today_duration, false) },
+
+        -- Since last resume
+        { _("Time spent reading this session"), datetime.secondsToClockDuration(user_duration_format, current_duration, false, true, true) },
+        { _("Pages read this session"), tonumber(current_pages), separator = true },
+
+        -- Today
+        { _("Time spent reading today ▸"), datetime.secondsToClockDuration(user_duration_format, today_duration, false, true, true),
```

Comment:
I knew this would be a thing... I copied the behavior from the menus to have it be unified. Not sure how else we can indicate that a row has more information upon tapping beyond what it's showing, besides the "(Tap to display)" which seems verbose. I honestly like the arrow... can't the RTL languages translate it to ◂? ;P

Reply:
While I always advocate not being too clever, this is probably a case where we want to say something like `if ltr then ▸ else ◂` outside of the string we offer to translators.

Reply:
Well, you could always provide the string without ▸to translators.
And concatenate ▸or ◂ to that translated strings, with may be some BD.wrap() around one or the other, or both :) (to be tested with arabic/hebrew to see what's needed and works :)

Reply:
Yup, that's exactly what I mean. ^_^

Reply:
Something like this?

```lua
    local more_arrow = BD.rtlUIText() and "◂" or "▸"
...
        { _("Time spent reading today") .. " " .. more_arrow, datetime.secondsToClockDuration(user_duration_format,
...
        { _("Days reading this book") .. " " .. more_arrow, tonumber(total_days),
```

Reply:
Yes for the concat. (And see if you need to BD.wrap() any of these 2 sides.)
Not sure about `BD.rtlUIText()`, which seems seldom used.
`BD.mirroredUILayout()` may be just ok.

Reply:
Looks good to my completely untrained eyes without the `BD.wrap()`... right?

![Reader_2022-12-12_033107](https://user-images.githubusercontent.com/10296053/207035089-f6bf1e16-6173-480e-bd6d-774a0da2a605.png)

Reply:
Yep, looks ok (both cases we should handle are on your screenshot)!

But there might be an issue with the hms duration format: I guess it shoud still look fully LTR, unless this is fine for Arabic, pinging @zeyadas: Does "s08m41h2" reads to you as "Z hours, 41 minutes and 8 seconds ?
Anyway, not for this PR to solve :)